website/src/routes/api/(schemas)/string/index.mdx

```diff
@@ -2,6 +2,116 @@
 title: string
 ---
 
+import { Property } from '~/components';
+import { Link } from '@builder.io/qwik-city';
+
 # string
 
-> The content of this page is not yet ready. Until then just use the [source code](https://github.com/fabian-hiller/valibot/blob/main/library/src/schemas/string/string.ts).
+Schema that parses the input to string.
+
+```ts
+const schema = string(errorMessage, pipe);
+```
+
+## Parameter
+
+- `errorMessage` <Property type='string'/> = 'Invalid type'
+- `pipe` <Property type={{ type: 'custom', name: 'Pipe<string>' }}/> = []
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)
I think `Pipe` doc is also needed.
Can I write this doc in another PR?

Reply:
Yes, but wait until I give you feedback on this PR.